.github/workflows/integrate.yaml

```diff
@@ -24,64 +24,47 @@ jobs:
     - name: Check flake8
       run: flake8
 
-  test:
-    name: Test
+  deploy:
+    name: Integration Test
     runs-on: ubuntu-latest
 
     steps:
     - name: Check out repo
       uses: actions/checkout@v2
 
-    - uses: balchua/microk8s-actions@v0.2.2
+      # TODO update with main actions-operator once Aggelos PR is merged
+    - name: Setup operator environment
+      uses: neoaggelos/actions-operator@attempt-fix-microk8s-1.19-issue
```

Comment:
If you use `charmed-kubernetes/actions-operator@main` you can get rid of a ton of boilerplate below, like installing charmcraft and setting up `kubectl`, though I'm not sure if you have to use this particular action and microk8s version.

Reply:
Great thanks for pointing this out. Since the version of microk8s is 1.19 I doubt it will install the latest candidate for charmcraft, so I'll remove everything except `sudo snap refresh charmcraft --channel latest/candidate`